app/views/layouts/_nav_menu.html.erb

```diff
@@ -12,7 +12,7 @@
     <div class="border-0 border-b-1 border-solid border-base-20 p-1">
       <a href="<%= dashboard_path %>" class="crayons-link crayons-link--block">Dashboard</a>
       <a href="<%= new_path %>" class="crayons-link crayons-link--block">Write a Post</a>
-      <a href="<%= readinglist_path %>" class="crayons-link crayons-link--block">Reading List</a>
+      <a href="<%= readinglist_path %>" class="crayons-link crayons-link--block">Reading list</a>
```

Comment:
I think we should keep the capitalization here, as all the other entries have it

Reply:
We don't have it in the main nav and I think our tendency is to keep only first letter capitalized.. But truth to be told - I don't care that much right now as I hope to rename that whole place to be "Saved" :D